test/specs/EventsSpec.js

```diff
@@ -22,96 +22,92 @@ describe("Event Tests", function() {
 
     describe("Mouse Events", function() {
 
-      it("'slideStart' event is triggered properly and can be binded to", function() {
+      it("'slideStart' event is triggered properly and can be binded to", function(done) {
```

Comment:
There are lots of tests that "pass", but have no expectations and these log errors show up.

![pull-902-no-expectations](https://user-images.githubusercontent.com/5477929/50370893-3208d000-057d-11e9-8bee-534bb9606da5.png)

Reply:
yea the reason i did this was that for these tests, there is nothing to really assert. i will see if there is an option we can pass to jasmine to suppress these

Reply:
actually...i can refactor these to use the spy pattern we are using below. that way we can assert if the spy was invoked or not

Reply:
Nice!